Cargo.toml

```diff
@@ -12,6 +12,8 @@ members = [
   "yew-router/examples/minimal",
   "yew-router/examples/router_component",
   "yew-router/examples/switch",
+  "yew-router/examples/servers/actix",
```

Comment:
these were intentionally left out because they prevent using cargo to target wasm. We should put them in their own workspace at some point :)

Reply:
Would putting them into their own workspace by adding `[workspace]` to their Cargo.toml work (as per latest push)? That allows running them without having to modify files.

Reply:
Yup, that works, thanks! Do you mind updating `yew/examples/server` with the same fix?

Reply:
Great. Sure, added to `yew/examples/server` as well.